### ISSUE TITLE
flow CLI panics when deploying contract via "flow accounts add-contract"

### ISSUE THREAD

**Issue by allenoooo** [onflow/flow-emulator#63]
### Instructions

Please fill out the template below to the best of your ability and include a label indicating which tool/service you were working with when you encountered the problem.

### Problem

Go panics when deploying a _specific contract_ via the CLI. When deploying _all contracts_ using the same config, etc, the contract is successfully deployed.

### Steps to Reproduce

When deploying a contract via `flow accounts add-contract Emit ./contract.cdc`, I get a panic:
```
❯ flow accounts add-contract Emit ./contract.cdc
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xc7ceb2]

goroutine 1 [running]:
github.com/onflow/flow-cli/pkg/flowkit.(*Account).Address(...)
        github.com/onflow/flow-cli/pkg/flowkit/account.go:40
github.com/onflow/flow-cli/pkg/flowkit.addAccountContractWithArgs(0x0, 0x7b2911503d08, 0x4, 0xc0003d2090, 0x83, 0xc000301a70, 0x0, 0x0, 0x7939b2, 0xc0000d6000, ...)
        github.com/onflow/flow-cli/pkg/flowkit/transaction.go:112 +0x272
github.com/onflow/flow-cli/pkg/flowkit.NewAddAccountContractTransaction(...)
        github.com/onflow/flow-cli/pkg/flowkit/transaction.go:80
github.com/onflow/flow-cli/pkg/flowkit/services.(*Accounts).AddContract(0xc0000c93b0, 0x0, 0x7b2911503d08, 0x4, 0xc00040e000, 0x83, 0x283, 0x0, 0x0, 0x0, ...)
        github.com/onflow/flow-cli/pkg/flowkit/services/accounts.go:223 +0xd7
github.com/onflow/flow-cli/internal/accounts.addContract(0xc00033c1e0, 0x2, 0x2, 0x16e5e80, 0xc00033a1f0, 0x0, 0x0, 0x1416e52, 0x4, 0x0, ...)
        github.com/onflow/flow-cli/internal/accounts/contract-add.go:67 +0x205
github.com/onflow/flow-cli/internal/command.Command.AddToParent.func1(0x1f2a9a0, 0xc00033c1e0, 0x2, 0x2)
        github.com/onflow/flow-cli/internal/command/command.go:116 +0x395
github.com/spf13/cobra.(*Command).execute(0x1f2a9a0, 0xc000142030, 0x2, 0x2, 0x1f2a9a0, 0xc000142030)
        github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc000318500, 0xc00085bf20, 0x1, 0x1)
        github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
        github.com/onflow/flow-cli/cmd/flow/main.go:69 +0x425
```
* Note that I get the same panic with or without the presence of a `--network=emulator` flag, and when I try to do anything that changes the state of the protocol _and_ is a `flow accounts...` subcommand. i.e:
```
❯ flow accounts get f8d6e0586b0a20c7

Address  0xf8d6e0586b0a20c7
Balance  9999999999.99500000
Keys     1

Key 0   Public Key               c1d20f3a7abd3663c154ce241807634502db84df45b362dd355976c5cb84c2d07595bec43d2fddad808d7957fc971d32048b7491461d300e618b979073ad73a0
        Weight                   1000
        Signature Algorithm      ECDSA_P256
        Hash Algorithm           SHA3_256
        Revoked                  false
        Sequence Number          3
        Index                    0

Contracts Deployed: 3
Contract: 'Emit'  // <----- This was deployed AFTER this report, as shown below.
Contract: 'FlowServiceAccount'
Contract: 'FlowStorageFees'


Contracts (hidden, use --include contracts)
```
works, while:
```
❯ flow accounts remove-contract Emit --network=emulator
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xc7cbc2]

goroutine 1 [running]:
github.com/onflow/flow-cli/pkg/flowkit.(*Account).Address(...)
        github.com/onflow/flow-cli/pkg/flowkit/account.go:40
github.com/onflow/flow-cli/pkg/flowkit.NewRemoveAccountContractTransaction(0x0, 0x7d9a37ceb625, 0x4, 0x0, 0x8, 0xc000187800)
        github.com/onflow/flow-cli/pkg/flowkit/transaction.go:89 +0x22
github.com/onflow/flow-cli/pkg/flowkit/services.(*Accounts).RemoveContract(0xc000342f60, 0x0, 0x7d9a37ceb625, 0x4, 0x0, 0x0, 0x0)
        github.com/onflow/flow-cli/pkg/flowkit/services/accounts.go:309 +0x85
github.com/onflow/flow-cli/internal/accounts.removeContract(0xc0000c6da0, 0x1, 0x2, 0x16e5e80, 0xc000333960, 0x0, 0x0, 0x1416e52, 0x4, 0x0, ...)
        github.com/onflow/flow-cli/internal/accounts/contract-remove.go:58 +0xbb
github.com/onflow/flow-cli/internal/command.Command.AddToParent.func1(0x1f2ac20, 0xc0000c6da0, 0x1, 0x2)
        github.com/onflow/flow-cli/internal/command/command.go:116 +0x395
github.com/spf13/cobra.(*Command).execute(0x1f2ac20, 0xc00018c030, 0x2, 0x2, 0x1f2ac20, 0xc00018c030)
        github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001fb180, 0xc0006bff20, 0x1, 0x1)
        github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
        github.com/onflow/flow-cli/cmd/flow/main.go:69 +0x425
```
doesn't.

When I get the panic, no request makes it to the gRPC server, so there are no relevant logs thrown by the emulator.

However, when deploying the same contract via: `flow project deploy --network=emulator`, I get the following result:

```
❯ flow project deploy --network=emulator

Deploying 1 contracts for accounts: one

Emit -> 0xf8d6e0586b0a20c7 (b396ee6e6e5a29a67407c56ba54077b3f6abed57e9549649184c487f7971707b)


✨ All contracts deployed successfully
```

The flow config:

```
❯ cat ~/flow.json

        │ File: /home/drifter/flow.json
   1   │ {
   2   │     "emulators": {
   3   │         "default": {
   4   │             "port": 3569,
   5   │             "serviceAccount": "one"
   6   │         }
   7   │     },
   8   │     "networks": {
   9   │         "emulator": "127.0.0.1:3569",
  10   │         "mainnet": "access.mainnet.nodes.onflow.org:9000",
  11   │         "testnet": "access.devnet.nodes.onflow.org:9000"
  12   │     },
  13   │     "accounts": {
  14   │         "one": {
  15   │             "address": "f8d6e0586b0a20c7",
  16   │             "key": "82b049f5fc2a963445213b47b7720a7ab92dd9366fdc0393c05959ec1bd43de4"
  17   │         },
  18   │         "two": {
  19   │             "address": "0x01cf0e2f2f715450",
  20   │             "key": "29ab6141b0746311f019fce9d9ea1e03f6793db1242e62b731847a8080b36051"
  21   │         },
  22   │         "three": {
  23   │             "address": "0x179b6b1cb6755e31",
  24   │             "key": "40dfa1ee8bdd6eb7daed35dc6aaa9bba2261e53916888c362e5d41b614bc8a77"
  25   │         }
  26   │     },
  27   │     "contracts": {
  28   │     "Emit": "/home/drifter/code/dapps/3mittr-contracts/3mit/contract.cdc"
  29   │   },
  30   │     "deployments": {
  31   │     "emulator": {
  32   │       "one": [
  33   │         "Emit"
  34   │       ]
  35   │     }
  36   │   }
  37   │ }
```

and the contract code:

```
❯ cat ~/code/dapps/3mittr-contracts/3mit/contract.cdc
        │ File: /home/drifter/code/dapps/3mittr-contracts/3mit/contract.cdc
   1   │ pub contract Emit {
   2   │     pub let emit: String
   3   │
   4   │     init() {
   5   │         self.emit = "test"
   6   │     }
   7   │
   8   │     pub fun getEmit(): String {
   9   │         return self.emit
  10  │     }
  11  │ }

```
### Acceptance Criteria
n/a

### Context
Not blocking anything. Low priority.

```
❯ pfetch
      /\          drifter@simulacrum
     /  \         os     Artix Linux
    /`'.,\        host   Blade 15 Base Model (Early 2020) - RZ09-0328 5.04
   /     ',       kernel 5.12.7-hardened1-1-hardened
  /      ,`\      uptime 8h 55m
 /   ,.'`.  \     pkgs   795
/.,'`     `'.\    memory 4201M / 15871M

```
```
❯ flow version
Version: v0.25.1
Commit: b7fe92a05d9d82496a2572e4245e85df335654b2
```
```
❯ flow status

Status:          🟢 ONLINE
Network:         emulator
Access Node:     127.0.0.1:3569
```
```
❯ procs flow
 PID:▲ User    │ TTY   CPU MEM CPU Time │ Command
               │       [%] [%]          │
 7750  drifter │ pts/5 0.0 0.2 00:00:01 │ flow emulator --persist -v
```

**Comment by allenoooo** [comment]
A community member in the flow discord messaged me about this the other night. seems that the issue is related to resource types not being present in the transaction. in the above case, I have renamed the default account to 'one'. if i add a `--signer` flag, i.e. 
```
❯ flow accounts remove-contract Emit --network=emulator --signer one
Transaction ID: 9e0986a2533bc0002d7c1995140c1ca0b571db2437c75c2459a3bc47bf6dce85
Contract Emit removed from account f8d6e0586b0a20c7.

Address  0xf8d6e0586b0a20c7
Balance  9999999999.99500000
Keys     1

Key 0   Public Key               c1d20f3a7abd3663c154ce241807634502db84df45b362dd355976c5cb84c2d07595bec43d2fddad808d7957fc971d32048b7491461d300e618b979073ad73a0
        Weight                   1000
        Signature Algorithm      ECDSA_P256
        Hash Algorithm           SHA3_256
        Revoked                  false
        Sequence Number          4
        Index                    0
```
the panic never happens, as you can see. i haven't looked at the code yet, but after i get my bearings a little more, i may take a stab at fixing the panic condition, if this issue is still open at that time.

**Comment by sideninja** [comment]
@allenoooo I can not reproduce this issue and I believe it was resolved in version 0.26.0, if that is true for you as well let me know so we can close the issue, however, your feedback opened another important issue you can read about it here: https://github.com/onflow/flow-cli/issues/346
Thank you for reporting!

**Comment by allenoooo** [comment]
@sideninja Indeed, on the new version, the panic doesn't happen anymore. Thanks!
